src/write/mod.rs

```diff
@@ -300,6 +300,20 @@ impl Object {
         offset
     }
 
+    /// Convert a symbol to a section offset.
+    ///
+    /// Returns an error if the symbol is not defined.
+    pub fn symbol_section_offset(&mut self, symbol_id: SymbolId) -> Result<(SymbolId, u64), ()> {
```

Comment:
`symbol_section_and_offset` is more descriptive. `symbol_section_offset` implies to me that only the offset is returned.